### DISCUSSION TITLE
Examples that use Protomaps / PMTiles to filter views of data on the same page?

### DISCUSSION THREAD

**Discussion by jaanaltosaar** [protomaps/PMTiles#386] [Q&A] [answered]
I finally got a baby Protomap working:

https://jaanli.github.io/new-york-real-estate/

Next I am trying to link this real estate data to demographics from the census: https://jaanli.github.io/american-community-survey/new-york-area/income-by-race

It would be nice to have the map change to show heatmaps of high- or low-prevalence self-reported race categories or income categories from census data (in the first link) as selections are made.

Open to any tips or in case folks are aware of examples! If this is not supported I'm happy to learn what it takes to extend the Protomaps library because it rocks 🚀

**Comment by bdon** [comment, marked as answer]
This is outside of the scope of pmtiles, you should join the datasets in tabular form before creating the tileset with tippecanoe (or other tilers) and then apply filtering using the MapLibre style language / feature state APIs.

**Reply by jaanaltosaar** [reply]
Will give this a shot!! Thank you so much 🤘🙇‍♂️
